src/twisted/web/test/test_static.py

```diff
@@ -330,7 +342,7 @@ def cbRendered(ignored):
             self.assertEqual(b''.join(request.written), content)
             self.assertEqual(
                 request.responseHeaders.getRawHeaders(b'content-length')[0],
-                networkString(str(len(content))))
+                str(len(content)).encode("utf-8"))
```

Comment:
Header values should be ASCII.

Reply:
This value will always be in the ascii range even when encodign with "utf-8".